dashboard/config/locales/blocks.en.yml

```diff
@@ -348,9 +374,8 @@ en:
             '"splatter"': Bubbles
             '"swirl"': Hypno
             '"snowflakes"': Snow
-            '"text"': Song Title
```

Comment:
Does it seem intentional that we removed Song Title from being translated?

Reply:
I think we removed this option from the block itself